.travis.yml

```diff
@@ -200,7 +200,7 @@ jobs:
           ${MAVEN_SKIP} -Dremoteresources.skip=true -Ddruid.generic.useDefaultValueForNull=${DRUID_USE_DEFAULT_VALUE_FOR_NULL}
         - sh -c "dmesg | egrep -i '(oom|out of memory|kill process|killed).*' -C 1 || exit 0"
         - free -m
-        - ${MVN} -pl ${MAVEN_PROJECTS} jacoco:report
+        - travis_wait 15 ${MVN} -pl ${MAVEN_PROJECTS} jacoco:report
```

Comment:
No, this is needed since `SuperSorterTest` runs for > 10 minutes sometimes and doesn't produce any log in a successful case. So, the wait for test to finish is increased to 15 minutes.